Description: Fix minor typo in src/kj/filesystem.c++
Forwarded: https://github.com/capnproto/capnproto/pull/746
Author: Thomas Lee <debian@tomlee.co>
Last-Update: 2018-09-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/kj/filesystem.c++
+++ b/src/kj/filesystem.c++
@@ -1511,7 +1511,7 @@
               entry.set(kj::mv(copy));
             } else {
               if (mode == TransferMode::MOVE) {
-                KJ_ASSERT(fromDirectory.tryRemove(fromPath), "could't move node", fromPath) {
+                KJ_ASSERT(fromDirectory.tryRemove(fromPath), "couldn't move node", fromPath) {
                   return false;
                 }
               }
@@ -1543,7 +1543,7 @@
               entry.set(kj::mv(copy));
             } else {
               if (mode == TransferMode::MOVE) {
-                KJ_ASSERT(fromDirectory.tryRemove(fromPath), "could't move node", fromPath) {
+                KJ_ASSERT(fromDirectory.tryRemove(fromPath), "couldn't move node", fromPath) {
                   return false;
                 }
               }
@@ -1560,7 +1560,7 @@
             // Since symlinks are immutable, we can implement LINK the same as COPY.
             entry.init(SymlinkNode { lastModified.orDefault(clock.now()), kj::mv(*content) });
             if (mode == TransferMode::MOVE) {
-              KJ_ASSERT(fromDirectory.tryRemove(fromPath), "could't move node", fromPath) {
+              KJ_ASSERT(fromDirectory.tryRemove(fromPath), "couldn't move node", fromPath) {
                 return false;
               }
             }
